test/config.test.ts

```diff
@@ -13,7 +13,7 @@ const basePath = "test/config";
 
 function assertSchema(name: string, userConfig: Config & { type: string }, tsconfig?: boolean) {
     return () => {
-        const config: Config = {
+        const config = {
```

Comment:
Why is this needed? If you don't want all parameters, use `Partial<Config>`.

Reply:
Yes, makes sense 👍

Reply:
Actually it looks like it can just be `Config`. I'll revert this.